docs/dev-guide/contributing/merging.rst

```diff
@@ -195,20 +142,19 @@ Infrequently, there may be a need to apply a change to an old release but not
 newer releases. This should only be a last resort.
 
 One way or another, it is important to merge this change into newer release
-branches, even if the actual changes don't get applied. When fixing code that no
-longer exists in newer branches, simply do the merge and resolve any conflicts
-that arise.
+branches, even if the actual changes don't get applied. When fixing code that
+no longer exists in newer branches, simply do the merge and resolve any
+conflicts that arise.
 
 Otherwise, to merge the work but not apply any of its code changes, use merge
 strategy "ours".
 
 ::
 
-  $ git merge -s ours username-bugfix
+  $ git merge -s ours my-important-fix
 
 In either case, git's history records that your fix has been applied to each
-release branch. Make sure the human-readable description of your fix accurately
-describes its scope. For example, a good commit message would be
-"Fixed memory use issue in ABC system, which was removed in pulp 2.1", or
-"Fixed a python 2.4 compatibility issue that is no longer applicable as of pulp
-2.2".
+branch. Make sure the human-readable description of your fix accurately
+describes its scope. For example, a good commit message would be "Fixed memory
```

Comment:
I think we should update our commit message suggestions, but I think that's a separate issue. I've created one for it [here](https://pulp.plan.io/issues/1321). No change here, but something I thought about as I read it.